### ISSUE TITLE
VS 2022 LNK1104 cannot open file 'gio-2glib-2.0.lib'

### ISSUE THREAD

**Issue by andynong** [wingtk/gvsbuild#808]
Apologies beforehand if this is not the appropriate question to ask here but I am not sure where else to best ask. I am trying GTK for the very first time and followed this windows install method which was clear and simple enough.

I have the GTK stack and am struggling with 'consuming' it into VS Studio.
After fumbling with VS project configuration property settings I managed to narrow down all the errors to just one Object File Library build link error:
<img width="669" alt="LNK1104" src="https://user-images.githubusercontent.com/122467324/211852413-7ed388a3-4a0d-4b1d-ada8-735b0f0753ea.png">

But I can not find said library within the gtk-build folder... am I missing something? I tried googling for solutions elsewhere to no avail. 

I tried both GTK 3.0 and GTK 4.0 to no success, after updating the paths IN VS accordingly I still get reduced to this one error.

The code I am running is simply the hello world code posted on the GTK site: [https://www.gtk.org/docs/getting-started/hello-world/](url)

**Comment by danyeaw** [comment]
Hi @andynong, did you try setting your environmental variables?

```
$env:Path = $env:Path + ";C:\gtk-build\gtk\x64\release\bin"
$env:LIB = "C:\gtk-build\gtk\x64\release\lib"
$env:INCLUDE = "C:\gtk-build\gtk\x64\release\include;C:\gtk-build\gtk\x64\release\include\cairo;C:\gtk-build\gtk\x64\release\include\glib-2.0;C:\gtk-build\gtk\x64\release\include\gobject-introspection-1.0;C:\gtk-build\gtk\x64\release\lib\glib-2.0\include;"
```

**Comment by andynong** [comment]
So in the end I learned how to compile it via the msys2 mingw64 way following another tutorial, using the CMD terminal to compile with GCC and that worked like a charm.

I did not add the LIB or INCLUDE variables that you mentioned, but now that I did the code compiles now in VS22 as well!
Thanks so much!

Now that I have both, is it okay to keep both? Will I run into any conflicts? From my understanding I have the non consumable files in msys2/mingw64 in my paths as well for gcc config purposes.

The other tutorial that I followed is: https://stdin.top/posts/gtk4-on-windows/ 
Ignoring the VS Code bits.

**Comment by danyeaw** [comment]
Hi @andynong, you can have both msys2 installed with a working GTK, and gvsbuild with a working GTK. They shouldn't conflict with each other because to use msys2 you need to open the MINGW-64 terminal.

Would you be interested in helping us improve our README with instructions on how to set the environmental variables?

**Comment by danyeaw** [comment]
Closed by f23cef91e2240de98d4b0669edb590fe2b832e41
